apps/verticalhome/js/configurator.js

```diff
@@ -20,6 +20,8 @@
   var singleVariantApps = {};
   var simPresentOnFirstBoot = true;
 
+  var DEFAULT_COLUMNS = 2;
```

Comment:
May be it is better to  use const instead var?
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Statements/const